### PR TITLE
Login user

### PR DESCRIPTION
**Pull request by 2HEADSs**
Fix user login functionality to work correctly, and return correct error if there is.

### CODE REVIEW

File: src/models/Organization.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ OrgnaizationSchema = new Schema({
   city: { type: String, required: true },
   //TODO:  minimum and maximum length address
   address: { type: String, required: true },
-  createdEvents: { type: ObjectId, ref: "Event", required: true },
+  createdEvents:  [{ type: ObjectId, ref: "Event", required: true }],
```
Comment (Getsov) [inline src/models/Organization.js:30]: Not required

File: src/models/Organization.js [hunk 2]
```diff
@@ -27,7 +27,9 @@ OrgnaizationSchema = new Schema({
...
+  //TODO: Will organization will like events
+  likedEvents: [{ type: ObjectId, ref: "Event", required: true }],
```
Comment (Getsov) [inline src/models/Organization.js:32]: Not required

File: src/models/Organization.js (hunk 1 above)
Comment (2HEADSs) [reply src/models/Organization.js]: Fixed and pushed the changes!

File: src/models/Organization.js (hunk 2 above)
Comment (2HEADSs) [reply src/models/Organization.js]: Fixed and pushed the changes!

**Review by Getsov — APPROVED** [review]

**Pull request merged**